frontend/amundsen_application/api/notice/v0.py

```diff
@@ -0,0 +1,64 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import json
+
+from http import HTTPStatus
+from typing import cast
+
+
+from flask import Response, jsonify, make_response, request, current_app as app
+from flask.blueprints import Blueprint
+from marshmallow import ValidationError
+from werkzeug.utils import import_string
+
+from amundsen_application.api.utils.request_utils import get_query_param
+from amundsen_application.base.base_notice_client import BaseNoticeClient
+
+LOGGER = logging.getLogger(__name__)
+NOTICE_CLIENT_INSTANCE = None
+
+notices_blueprint = Blueprint('notices', __name__, url_prefix='/api/notices/v0')
+
+
+def get_notice_client() -> BaseNoticeClient:
+    global NOTICE_CLIENT_INSTANCE
+    if NOTICE_CLIENT_INSTANCE is None and app.config['NOTICE_CLIENT'] is not None:
+        notice_client_class = import_string(app.config['NOTICE_CLIENT'])
+        NOTICE_CLIENT_INSTANCE = notice_client_class()
+    return cast(BaseNoticeClient, NOTICE_CLIENT_INSTANCE)
+
+
+@notices_blueprint.route('/table', methods=['GET'])
+def get_table_notices_summary() -> Response:
+    global NOTICE_CLIENT_INSTANCE
+    try:
+        client = get_notice_client()
+        if client is not None:
+            return _get_table_notices_summary_client()
+        payload = jsonify({'placeholder': 'this feature is not implemented'})
```

Comment:
for consistency, to have the same payload format as the other places

```suggestion
        payload = jsonify({'notices': {}, 'msg': 'This feature is not implemented'})
```